compiler/rustc_span/src/source_map.rs

```diff
@@ -753,9 +753,9 @@ impl SourceMap {
         }
     }
 
-    /// Given a 'Span', tries to tell if the next character is '>'
-    /// and the previous charactoer is '<' after skipping white space
-    /// return true if wrapped by '<>'
+    /// Given a 'Span', tries to tell if the next character is '>' or ')'
+    /// and the previous charactoer is '<' or '(' after skipping white space
+    /// return true if wrapped by '<>' or '()'
     pub fn span_wrapped_by_angle_bracket(&self, span: Span) -> bool {
```

Comment:
Change this name to something that better represents the functionality and add documentation to say what this should be used for.

Reply:
done.